.github/workflows/node-ci.yml

```diff
@@ -295,3 +317,17 @@ jobs:
                       Oh noes!! We couldn't find any changesets in this PR (${{
                       steps.short-sha.outputs.short_sha }}). As a result, we did not
                       publish an npm snapshot for you.
+
+            - name: Create or update npm snapshot comment - failure, concurrent with release
+              if: always() && steps.publish-snapshot.outputs.npm_snapshot_tag == '' && steps.check-release.outputs.release-count != '0'
```

Comment:
**note:** We want to update the PR comment if we never even published, so we always run this step.